templates/search.html

```diff
@@ -0,0 +1,169 @@
+                                    </small>
+                                    {% if form.class_code.errors %}
+                                        <div class="invalid-feedback">
+                                            {% for error in form.class_code.errors %}
+                                                <i class="fas fa-exclamation-circle me-1"></i>{{ error }}
+                                            {% endfor %}
+                                        </div>
+                                    {% endif %}
+                                </div>
+                                <div class="col-md-5" id="courseTypeContainer">
+                                    {{ form.course_type.label(class="form-label") }}
+                                    {{ form.course_type(class="form-select" + (" is-invalid" if form.course_type.errors else "")) }}
+                                    {% if form.course_type.errors %}
+                                        <div class="invalid-feedback">
+                                            {% for error in form.course_type.errors %}
+                                                <i class="fas fa-exclamation-circle me-1"></i>{{ error }}
+                                            {% endfor %}
+                                        </div>
+                                    {% endif %}
+                                </div>
+                            </div>
+                        </div>
+
+                        <!-- Term Section -->
+                        <div class="form-section">
+                            <h4><i class="fas fa-calendar-alt"></i>Term</h4>
+                            <div class="row g-4">
+                                <div class="col-md-12">
+                                    {{ form.term.label(class="form-label") }}
+                                    {{ form.term(class="form-select" + (" is-invalid" if form.term.errors else "")) }}
+                                    {% if form.term.errors %}
+                                        <div class="invalid-feedback">
+                                            {% for error in form.term.errors %}
+                                                <i class="fas fa-exclamation-circle me-1"></i>{{ error }}
+                                            {% endfor %}
+                                        </div>
+                                    {% endif %}
+                                </div>
+                            </div>
+                            <div class="mt-3">
+                                <div class="alert alert-info" role="alert">
+                                    <i class="fas fa-info-circle me-2"></i>
+                                    <strong>Note:</strong> Select a specific term or "All Terms" to include all available semesters. The default is the upcoming semester (Fall 2025).
```

Comment:
Do not hardcode the semester, it can be confusing.
```suggestion
                                    <strong>Note:</strong> Select a specific term or "All Terms" to include all available semesters. The default is the upcoming semester.
```

Reply:
Should I also send, DEAULT_TERM from my route and include that here? Or should we keep it clean and simple?